sdk/src/androidTest/java/com/hcaptcha/sdk/HCaptchaTestHtml.java

```diff
@@ -0,0 +1,46 @@
+package com.hcaptcha.sdk;
+
+class HCaptchaTestHtml implements IHCaptchaHtmlProvider {
+
+    @Override
+    public String getHtml() {
+        return "<html>\n"
+                + "<head>\n"
+                + "    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1, user-scalable=no\"/>\n"
+                + "</head>\n"
+                + "<body>\n"
```

Comment:
Would it be possible to keep the html in a `.html` file and package it in correctly at build time? Maybe via gradle tasks. This would ease the maintenance of the html and allow to benefit from IDE understanding the html code. Maybe as a follow up to not block current PR. @CAMOBAP